java-library/pom.xml

```diff
@@ -14,6 +14,10 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.plugin.version>3.13.0</maven.compiler.plugin.version>
+        <maven.source.plugin.version>3.3.1</maven.source.plugin.version>
+        <maven.javadoc.plugin.version>3.10.0</maven.javadoc.plugin.version>
+        <maven.surefire.plugin.version>3.5.0</maven.surefire.plugin.version>
```

Comment:
nit: it will be good to format these lines and align them.

Use `Alt+Shift+F` in VS Code for auto formatting of the file, it will prompt to install a compatible extension if you already don't have one